wasm/tools/fixtures/runner.py

```diff
@@ -276,52 +331,69 @@ def do_assert_unlinkable(command, store, module, all_modules, registered_modules
 
     if command.file_path:
         with pytest.raises(Unlinkable):
-            instantiate_module_from_wasm_file(command.file_path, store, registered_modules)
+            instantiate_module_from_wasm_file(command.file_path, runtime)
     else:
         raise Exception("Unhandled")
 
 
-def do_register(command, store, module, all_modules, registered_modules):
+def do_register(command: Register,
+                module: CurrentModule,
+                all_modules: AllModules,
+                runtime: Runtime) -> None:
     if command.name is None:
-        registered_modules[command.as_] = module
+        if module is None:
+            raise Exception("Invariant")
+        runtime.register_module(command.as_, module)
     else:
-        registered_modules[command.as_] = all_modules[command.name]
+        runtime.register_module(command.as_, all_modules[command.name])
 
 
-def do_action_command(command, store, module, all_modules, registered_modules):
-    run_opcode_action(command, store, module, all_modules, registered_modules)
+def do_action_command(command: TActionCommands,
+                      module: CurrentModule,
+                      all_modules: AllModules,
+                      runtime: Runtime) -> None:
+    run_opcode_action(command, module, all_modules, runtime)
```

Comment:
It confused me that this method exists. I guess just for the `do_*` consistency? It made me wonder if I was missing something else it did. Maybe it can be replaced with an alias, like:
```py
do_action_command = run_opcode_action
```